core/src/main/java/org/axonframework/commandhandling/model/LockingRepository.java

```diff
@@ -82,6 +83,21 @@ protected LockingRepository(Class<T> aggregateType, ParameterResolverFactory par
         this(aggregateType, new PessimisticLockFactory(), parameterResolverFactory);
     }
 
+    /**
+     * Initialize a repository with a pessimistic locking strategy and a parameter resolver factory.
+     *
+     * @param aggregateType            The type of aggregate stored in this repository
+     * @param parameterResolverFactory The parameter resolver factory used to resolve parameters of annotated handlers
+     * @param handlerDefinition        The handler definition used to create concrete handlers
+     */
+    protected LockingRepository(Class<T> aggregateType, ParameterResolverFactory parameterResolverFactory,
```

Comment:
Nit: same small thing I've been pointing out earlier, but I think it would be more consistent throughout if all the parameters have there own line if all of them do not fit on one line.

Additionally, I see that happening throughout all the 1600+ lines changed. I'll refrain on commenting on all of them, as that would be a bit much.

Reply:
This is automatic formatting by intelliJ which I was relying on.

Reply:
That makes full sense, I do that too. Do you think we should adjust the formatting settings a little? :thinking:

Reply:
While we're on this topic, I would suggest to do so. And, I would like to have a stylecheck rules which reflects our codacy configuration.

Reply:
Sounds good to me